examples/tv-casting-app/tv-casting-common/include/CHIPProjectAppConfig.h

```diff
@@ -61,3 +57,12 @@
 #define CHIP_ENABLE_ROTATING_DEVICE_ID 1
 
 #define CHIP_DEVICE_CONFIG_ROTATING_DEVICE_ID_UNIQUE_ID_LENGTH 128
+
+// Disable this since it should not be enabled for production setups
+#define CHIP_DEVICE_CONFIG_ENABLE_TEST_SETUP_PARAMS 0
+
+#define CHIP_DEVICE_CONFIG_DYNAMIC_ENDPOINT_COUNT 4
+
+// Include the CHIPProjectConfig from config/standalone
+// Add this at the end so that we can hit our #defines first
+#include <CHIPProjectConfig.h>
```

Comment:
I think we were already following the pattern of defining a macro only if it is not defined in this file? Why do we still need to put this at the bottom? I'd rather not rely on the relative ordering of the defines here. If there are some cases that don't follow the right pattern, I'd much rather have them fixed.

Reply:
The same configs are defined in `#CHIPProjectConfig.h` these don't get respected, we can either remove the #include altogether (that might cause us to miss stuff added later on)

The `CHIP_DEVICE_CONFIG_ENABLE_TEST_SETUP_PARAMS` should be set and by default it is set to 1